keras_cv/models/stable_diffusion/stable_diffusion.py

```diff
@@ -22,11 +22,16 @@
 """
 
 import math
+from typing import List
```

Comment:
lets keep type annotations unified across the file - and thus far we don't have them.

Reply:
Fixed!